packages/components/src/button/index.stories.tsx

```diff
@@ -0,0 +1,107 @@
+import { fn } from '@storybook/test';
+import { Button } from '.';
+import type { Meta, StoryObj } from '@storybook/react';
+
+/**
+ * This button has been deprecated due to aggressive and generic CSS that breaks many other buttons when imported.
+ * Use the [`Button` component](https://wordpress.github.io/gutenberg/?path=/docs/components-button--docs)
+ * from `@wordpress/components` instead.
+ */
+const meta: Meta< typeof Button > = {
+	title: 'packages/components/Button',
+	component: Button,
+	args: {
+		onClick: fn(),
```

Comment:
Is this the recommended way to represent a callback in Storybook args / controls? TIL

Reply:
[Yes](https://storybook.js.org/docs/essentials/actions#action-args), I guess it kind of got closer to Jest.